front/package.json

```diff
@@ -13,7 +13,7 @@
     "initdb": "npx tsx admin/db.ts"
   },
   "dependencies": {
-    "@dust-tt/sparkle": "^0.2.73",
+    "@dust-tt/sparkle": "^0.2.75",
```

Comment:
Updating both sparkle & front in the same PR cannot be done unfortunately, first we need to do sparkle, then front

Reply:
@philipperolet it's not the case, Sparkle 75 has already been merged and deployed.

Reply:
If it was, we wouldn't see the changes in the PR so there's something to dig here, let's check